packages/repository/src/__tests__/fixtures/mixins/find-by-title-repo-mixin.ts

```diff
@@ -4,26 +4,31 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {Constructor} from '@loopback/context';
-import {CrudRepository, Entity, Where} from '../../..';
+import {CrudRepository, Model, Where} from '../../..';
 
-export interface FindByTitle<E extends Entity> {
+/**
+ * An interface to allow finding notes by title
+ */
+export interface FindByTitle<E extends Model> {
```

Comment:
```suggestion
export interface FindByTitle<M extends Model> {
```

I find it weird to use `E` for models, I would expect `E` to be used for entities.

The same comment applies to other places using `E` too.